packages/react/src/components/MessageBar/MessageBar.types.ts

```diff
@@ -128,6 +128,12 @@ export interface IMessageBarProps extends React.HTMLAttributes<HTMLElement>, Rea
    * @default true
    */
   delayedRender?: boolean;
+
+  /**
+   * An optional override to show the expand/collapse icon.
```

Comment:
```suggestion
   * An optional override to show the expand/collapse icon. It will only be shown by default for single-line truncated MessageBars that do not have actions.
```

Your call on if you think the extra wording is worth it or too much, my thought here was since we didn't really document the default behavior anywhere else, it wouldn't hurt to do it here